app/javascript/src/components/icon/Expand.svelte

```diff
@@ -0,0 +1,51 @@
+<script>
+  export let contract = false
+
+  let cssRotation = ""
+  $: {
+    if (contract) cssRotation = "rotate(180deg)"
+    else cssRotation = ""
+  }
+</script>
+
+<style>
+  .arrow {
+    fill: currentColor;
+    transform-origin: center;
+    transform-box: fill-box; /* transform-origin relative to this element, not the <svg> */
+  }
+</style>
+
+<svg
+  class={$$restProps.class || ""}
+  style={$$restProps.style || ""}
```

Comment:
This could a spread, otherwise you might as well add them as proper props. The `|| ""` is also unnecessary, when it's null the property won't show up at all which is a little nicer. 

```suggestion
  {...$$restProps}
```